base/complex.jl

```diff
@@ -541,6 +541,30 @@ function cis(z::Complex)
     Complex(v * c, v * s)
 end
 
+cispi(theta::Real) = Complex(reverse(sincospi(theta))...)
+
+"""
+    cispi(z)
+
+Return ``\\exp(iπz)``.
+
+# Examples
+```jldoctest
+julia> cispi(1) == -1
```

Comment:
I think it would be better to just print the output of `cispi` here. Maybe also add an example with complex input?